src/pipx/interpreter.py

```diff
@@ -41,27 +43,63 @@ def __init__(self, source: str, version: str, wrap_message: bool = True):
                 message += "The provided version looks like a path, but no executable was found there."
             if potentially_pylauncher:
                 message += (
-                    "The provided version looks like a version for Python Launcher, " "but `py` was not found on PATH."
+                    "The provided version looks like a version, "
+                    "but both the python command and the Python Launcher were not found on PATH."
                 )
         if source == "the python-build-standalone project":
             message += "listed in https://github.com/indygreg/python-build-standalone/releases/latest."
         super().__init__(message, wrap_message)
 
 
+def find_unix_command_python(python_version: str) -> Optional[str]:
+    try:
+        parsed_python_version = version.parse(python_version)
+    except version.InvalidVersion:
+        logger.info(f"Invalid Python version: {python_version}")
+        return None
+    else:
```

Comment:
Is this needed?

Reply:
Thanks for pointing out this overshooting.